tests/test_store.py

```diff
@@ -956,27 +1097,40 @@ def test_add_annotations_to_entities_files(
             expected_filenames (list(str)): expected list of file names
             expected_entity_ids (list(str)): expected list of entity ids
         """
+        async def mock_format_row_annos():
+            await asyncio.sleep(0.1)
+        
+        async def mock_process_store_annos(requests):
+            await asyncio.sleep(0.1)
+
         with patch(
             "schematic.store.synapse.SynapseStorage.getFilesInStorageDataset",
             return_value=files_in_dataset,
         ):
-            manifest_df = pd.DataFrame(original_manifest)
+            with patch('schematic.store.synapse.SynapseStorage.format_row_annotations', return_value=mock_format_row_annos, new_callable=AsyncMock) as mock_format_row:
```

Comment:
I recommend to update your precommit black config to include these files as well:

https://github.com/Sage-Bionetworks/schematic/blob/develop/.pre-commit-config.yaml#L13

Reply:
Thanks. Will be resolved in PR: https://github.com/Sage-Bionetworks/schematic/pull/1442